api/src/wfl/api/handlers.clj

```diff
@@ -78,7 +78,9 @@
         (mapv (partial go! tx)
           (if-let [uuid (-> request :parameters :query :uuid)]
             [(workloads/load-workload-for-uuid tx uuid)]
-            (workloads/load-workloads tx)))))))
+            (if-let [project (-> request :parameters :query :project)]
+              [(workloads/load-workload-for-project tx project)]
+              (workloads/load-workloads tx))))))))
```

Comment:
I personally don't like the nested if here, but after trying out multimethod, protocols and `case`, I feel like this is still the most straightforward and not overkilling way to achieve the goal, so I leave this as-is.

Reply:
Agreed. Would be nice if there was a `cond-let` or something!